Troubleshooting.md

```diff
@@ -18,6 +18,11 @@ The debugger works with [Language Support for Java(TM) by Red Hat](https://marke
 
 ## Failed to resolve classpath:
 ### Reason:
+Below are the common failure reasons.

```

Comment:
In the document, it should avoid using `xyz` which is not meaningful. 

Should use meaningful name such as `mainClassName`, `projectName`

Reply:
Nice suggestion. thanks.